Mathlib/Data/Finset/Image.lean

```diff
@@ -78,6 +78,11 @@ theorem mem_map_equiv {f : α ≃ β} {b : β} : b ∈ s.map f.toEmbedding ↔ f
 theorem mem_map' (f : α ↪ β) {a} {s : Finset α} : f a ∈ s.map f ↔ a ∈ s :=
   mem_map_of_injective f.2
 
+@[simp 1100]
+theorem mem_map_mk (f : α → β) {a : α} {s : Finset α} (hf : Function.Injective f) :
```

Comment:
```suggestion
theorem mem_map_mk {f : α → β} {a : α} {s : Finset α} (hf : Function.Injective f) :
```
I'd also personally prefer `s` to be explicit, but that's up to you

Reply:
This is copied from the lemma above too